components/Profile/ProfileContainer.tsx

```
@@ -1,99 +1,67 @@
-
-  const positionsByMarketTicker = aggregatePositionsByMarketTicker();
+  const { aggregatedPositions, sumPnL } = useUserPositions(ss58Address);
 
   const { data: h160address } = useTanstackQuery({
     queryKey: ['h160', ss58Address],
     queryFn: addressMatcherApi.geth160Address,
   });
 
-  const { displayName } = useDisplayName(ss58Address);
+  const { data: balanceData, refetch } = useNativeCurrencyBalance(
+    h160address && h160address.data.ethAddress
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)